doc/usecases/core/UC_keyset_index_access_many.md

```diff
@@ -0,0 +1,26 @@
+# Use Case: Index access to `KeySet`
+
+## Summary
+
+- **Scope:** `libelektra-core`
+- **Level:** Developer Goal
+- **Actors:** Core, Caller
+- **Brief:** Caller accesses contents of `KeySet` by index
+
+## Scenarios
+
+- **Precondition:**
+  - [`KeySet` has been created](UC_keyset_create.md).
+  - [`Key`(s) have been inserted into `KeySet`](UC_keyset_insert.md).
+- **Main success scenario:**
+  - Caller requests `Key`s in valid index range (`[f, l)` with `0 <= f < size && f <= l <= size`) from `KeySet` KS
```

Comment:
This description doesn't motivate why the caller needs this. Probably an example would help.

Reply:
See https://github.com/ElektraInitiative/libelektra/pull/4797#discussion_r1064486145

Reply:
Also on a second read it is quite difficult for me to relate to the text as a use case. It reads more like a description of semantics of a function than a use case.